### ISSUE TITLE
Issue with "Using the alertdialog role": Missing content

### ISSUE THREAD

**Issue by Ryuno-Ki** [mdn/content#4319]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_alertdialog_role

#### What information was incorrect, unhelpful, or incomplete?

- No sidebar
- „TBD” in Working Examples
- „TBD” instead of Compatibility table => Collaborate with https://a11ysupport.io/ ? CC @accessibilitysupported @aardrian 
- Empty „Additional resources”

#### Specific section or headline?

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_alertdialog_role#examples onwards

#### What did you expect to see?

Content :heart_eyes_cat: 

#### Did you test this? If so, how?

I opened it in Firefox on Sabayon Linux

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/accessibility/aria/aria_techniques/using_the_alertdialog_role`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_alertdialog_role
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/accessibility/aria/aria_techniques/using_the_alertdialog_role/index.html
* Last commit: https://github.com/mdn/content/commit/72e0d271f16ce9d0c4389b9f525fa81e6df9c5f5
* Document last modified: 2021-02-19T18:53:08.000Z

</details>

**Comment by estelle** [comment]
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alertdialog_role has been added.
